S2VX.Game/Timeline.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using osu.Framework.Allocation;
+using osu.Framework.Extensions.Color4Extensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osuTK.Graphics;
+using SixLabors.ImageSharp;
+using osuTK;
+using SixLabors.ImageSharp.Processing;
+using osu.Framework.Graphics.UserInterface;
+using osu.Framework.Text;
+using System.Linq;
+
+namespace S2VX.Game
+{
+    public class Timeline : CompositeDrawable
+    {
+        [Cached]
+        private TimelineSlider TimelineSlider = new TimelineSlider
+        {
+            Name = "Slider",
+            Colour = Color4.White,
+            Height = slider_height,
+            Width = timeline_width / 150, //600
+        };
+
+        private const float timeline_height = 0.075f;
+        private const float timeline_width = 1.0f;
+        private const float slider_height = 0.5f;
+        private Container LineContainer { get; set; }
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            RelativeSizeAxes = Axes.Both;
+            InternalChildren = new Drawable[]
+            {
+                LineContainer = new Container
+                {
+                    Name = "Timeline content",
```

Comment:
Check if this really needed, rather not make properties if we aren't going to use them, maybe it's used for debug?

Reply:
We'll try and keep these going forward since it helps nicely with debug